test/bun.js/jest.test.js

```diff
@@ -0,0 +1,43 @@
+import { beforeAll, it, describe, expect, afterAll } from 'bun:test';
+
+var before = false;
+var after = false;
+
+beforeAll(async () => {
+  if (before) throw new AssertionError("before should be false");
+  if (after) throw new AssertionError("after should be false");
+  await new Promise(resolve => setTimeout(() => resolve(before = true), 1000));
+});
+
+it("should run after beforeAll()", async () => {
+  expect(before).toBe(true);
+  expect(after).toBe(false);
+});
+
+afterAll(() => {
```

Comment:
we have tests for the non-async versions here https://github.com/oven-sh/bun/blob/b86915c80819e61e6e1932d7802a4c31c73094e9/test/bun.js/bun-test/jest-hooks.test.ts#L9

Reply:
Ah okay, I'll lump these tests together then.